index.html

```diff
@@ -95,7 +129,9 @@
         </div>
 
         <div class="project2">
-          <div class="project2-img"></div>
+          <div class="project2-img">
+            <img src="assests/images/project2-img.svg" alt="project1-img" />
+          </div>
           <div class="project2-content">
             <div class="project2-content-sec1">
               Multi-Post Stories <br />
```

Comment:
This is a project title so kindly use a header tag such as h2 - h6 to make your work semantically correct.